jax/_src/named_sharding.py

```diff
@@ -108,6 +108,9 @@ class NamedSharding(JSharding.Sharding):
   Args:
     mesh: A :class:`jax.sharding.Mesh` object.
     spec: A :class:`jax.sharding.PartitionSpec` object.
+    memory_kind: A string indicating the memory kind of the sharding. Note that
```

Comment:
can you not mention the ShapedArray thing? That seems unrelated to NamedSharding docstring.